src/lib/ggpo/pevents.h

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
I think you only need this to implement `Poll`.  IMO, it would be better to move the functionality of `Poll` into `Platform` and write a POSIX native implementation rather than mapping to the  Win32 API.